backend/socialdistribution/settings.py

```diff
@@ -149,3 +149,7 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Global variables
+HOST_URL = 'http://127.0.0.1:8000/'
```

Comment:
Should this be an environment variable? When deployed the host will be different right?

Reply:
`request.build_absolute_uri() ` gives the current url, extract host from there

Reply:
> Should this be an environment variable? When deployed the host will be different right?

Yes

Reply:
> `request.build_absolute_uri() ` gives the current url, extract host from there

There is a more specific method called request.get_host, but for some reason not working, so I just manually write the host.

Reply:
I would say make it an `env` variable and then we dont need to worry about getting it from the method if it isnt working, otherwise I am good with the PR

Reply:
I have moved it to env and edited readme, now we all require a new variable "HOST_URL" in .env